hal/network/ncp_client/sara/sara_ncp_client.cpp

```diff
@@ -1092,6 +1097,21 @@ int SaraNcpClient::selectNetworkProf(ModemState& state) {
         CHECK(checkNetConfForImsi());
     }
 
+    if (ncpId() == PLATFORM_NCP_SARA_R510) {
```

Comment:
This check is because we support other non-R510 SARA modems in the 4.x release line, right? Specifically this is the U260?